cmd/azurerm.go

```diff
@@ -68,12 +68,12 @@ var (
 
 			if hclOut != nil {
 				logger.Log("msg", "initializing HCL writer")
-				hclW = hcl.NewWriter(hclOut)
+				hclW = hcl.NewWriter(hclOut, &writer.Options{Interpolate: !viper.GetBool("no-interpolate")})
```

Comment:
For all this changes you should define the Options before and just pass it, no initialize it each time.